genestack_client/files_util.py

```diff
@@ -642,27 +642,4 @@ def collect_metainfos(self, accessions):
         :return: list of metainfo objects
         :rtype: list[Metainfo]
         """
-        return map(Metainfo.parse_metainfo_from_dict, self.invoke('getMetainfo', accessions))
-
-    def create_dataset(self, name, dataset_type, children, parent=None):
```

Comment:
Looks like this breaks backward compatibility with existing scripts, is it planned?

Reply:
Need to return this method back in next way:

- print deprecation warning to stderr
- try to call new python method  (catch genestack exception)
- invoke old method

This will allow to use new version of python client with dotorg server